### PR TITLE
Adds a bunch of stuff to club, final attempt

### PR DESCRIPTION
**Pull request by genessee-forgot-his-password-again**
## About The Pull Request

- Adds 6 new club-themed disks! 2 are used for printing various utensils, 2 are used for brewing your own drinks via the power of the autolathe, 1 is used for making your own packaged food, and 1 has a mix of both brewing and food-packing.
- Adds an autolathe to the kitchen for club workers to actually use the disks they have.
- Adds 3 of the 6 disks to spawn roundstart at club - the Utensil-printing disk, the Glassware-printing disk, and the basic brew/food-packing disk. Spares of each disk can be found in the Club Manager closet.
- Gives a bunch of stuff in the code matter values so they can be printed in autolathes. Namely, most booze bottles and cans now can be recycled.
- Adds a new codersprite disk for the 'club' faction. I'm open to change it if an actual spriter steps up and improves it.

Alcohol generally requires glass (for the bottle) as well as a mix of ethanol, water, and sometimes a third ingredient for more potent brews. 
Soda generally requires steel (for the can) as well as a mix of sugar, soda water, and usually a third ingredient for flavor.
Food just requires some plastic for the packaging plus some nutriment, protein, or sugar, depending on what you're making.

bar1
disc1

I don't know why it's saying I'm changing 600000 lines, i think it has to do with the fact i initially fucked up my mapmerge - it was at 300000 then and all red - and then i did it right and now it's saying it adds 300000 and takes away 300000

i have no idea how to fix it and it's 6am

## Why It's Good For The Game

I thought it would be really cool to have it so club could make their own packaged food for sale and transport, and for them to be able to brew their own basic booze with the autolathe. This would also make it feasible for vagabonds to make their own booze stands if they can get the raw ethanol.

Plus, I think that doing something new with the autolathe system - combining and bottling liquids, in this case - is really cool and helps push the 'eris has the cool new things' deal that eris has

## Changelog
:cl:
add: Club Kitchen now has an autolathe and 3 new disks roundstart
add: 6 new club-themed disks that print utensils, food, and drink
tweak: Many bottles and cans, as well as some packaged foods, now have material values and can be recycled
/:cl:

### CODE REVIEW

**Comment by zombiemimic** [comment]
I think the rarity values are the wrong way around: lower values are more common

File: code/datums/autolathe/containers.dm [hunk 1]
```diff
@@ -59,3 +59,167 @@
...
+	
+/datum/design/autolathe/container/red_mug
+	name = "red coffee mug"
+	build_path = /obj/item/weapon/reagent_containers/food/drinks/mug/red
+	
+/datum/design/autolathe/container/metal_mug
+	name = "metal coffee mug"
+	build_path = /obj/item/weapon/reagent_containers/food/drinks/mug/metal
+	
+/datum/design/autolathe/container/metal_mug
```
Comment (Firefox13) [inline code/datums/autolathe/containers.dm:93]: Duplicate

File: code/datums/autolathe/food.dm [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+	build_path = /obj/item/weapon/reagent_containers/food/snacks/tastybread
+	chemicals = list(
+	"nutriment" = 6
+	)
+
+/datum/design/autolathe/food/packaged/liquidfood
+	build_path = /obj/item/weapon/reagent_containers/food/snacks/liquidfood
+	chemicals = list(
+	"nutriment" = 20 // the 3u of iron is freeeeeee! grinding the mre destroys the trash so they lose the 1 plastic they used to make it, so its fair nuff
+	)
```
Comment (Firefox13) [inline code/datums/autolathe/food.dm:43]: Newline

File: code/game/objects/items/weapons/design_disks/club.dm [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+	license = 10 // should be more than enough to supply someone for a full round
+	designs = list(
+	/datum/design/autolathe/food/packaged/candybar = 0, // free because it's also free in the basic bar disk
+	/datum/design/autolathe/food/packaged/chips = 0,
+	/datum/design/autolathe/food/packaged/cheesiehonkers = 0,
+	/datum/design/autolathe/food/packaged/breadtube,
+	/datum/design/autolathe/food/packaged/raisin,
+	/datum/design/autolathe/food/packaged/jerky,
+	/datum/design/autolathe/food/packaged/liquidfood = 2 // its got a lot of nutriment so it's worth 2 license points
+	)
```
Comment (Firefox13) [inline code/game/objects/items/weapons/design_disks/club.dm:109]: Newline

**Review by Firefox13 — COMMENTED** [review]
There's three hundred thousands of changes because the map isn't mapmerged and not in TGM format, use mapmerge2 tool
The disks shouldn't appear in maints because it can trivialize the club out of food chain and I dislike their whole idea at all since you can just fucking steal them and leave club boned.
And they already start with an autolathe in their backroom downstairs so there's no need to add another one.
Half of the code comments are off-topic or explain obvious things

> Pull requests should not contain commented code except TODOs and explanation comments.

File: code/datums/autolathe/containers.dm (hunk 1 above)
Comment (genessee-forgot-his-password-again) [reply code/datums/autolathe/containers.dm]: Whoops, fixed

File: code/datums/autolathe/food.dm (hunk 2 above)
Comment (genessee-forgot-his-password-again) [reply code/datums/autolathe/food.dm]: Fixed, sorry

File: code/game/objects/items/weapons/design_disks/club.dm (hunk 3 above)
Comment (genessee-forgot-his-password-again) [reply code/game/objects/items/weapons/design_disks/club.dm]: Fixed, didn't know I needed newline in stuff

**Comment by genessee-forgot-his-password-again** [comment]
> There's three hundred thousands of changes because the map isn't mapmerged and not in TGM format, use mapmerge2 tool

yeah i always have trouble with mapmerging, i'll look into fixing it ASAP

> The disks shouldn't appear in maints because it can trivialize the club out of food chain

i disagree since imo club is already trivialized out of the food chain since you can find lots of food in junk piles + they barely make food at all since cooking is really annoying on eris

also they do need nutriment and stuff to actually make, so they don't really make their own food - they just package it for ease of transport

> and I dislike their whole idea at all since you can just fucking steal them and leave club boned.

also disagree because this doesn't remove club's ability to cook food normally and mix drinks normally, this just gives them a way to make lots of packaged food / replenish their beer stocks easily -- if they get stolen it makes club's life harder but they aren't screwed

also they have a spare set of disks in the Club Manager office if the first set does get stolen

**Comment by genessee-forgot-his-password-again** [comment]
> Half of the code comments are off-topic or explain obvious things
> 
> > Pull requests should not contain commented code except TODOs and explanation comments.

Cut down on the comments a ton, let me know if anything else sticks out as odd on that front

**Comment by Firefox13** [comment]
> since you can find lots of food in junk piles

#6149

**Comment by genessee-forgot-his-password-again** [comment]
> > since you can find lots of food in junk piles
> 
> #6149

alright fair enough, but my point still stands -

this does not create food, this simply takes existing nutriment and turns it into a packaged food

**Comment by genessee-forgot-his-password-again** [comment]
> I think the rarity values are the wrong way around: lower values are more common

fixed, thanks for letting me know :)

**Comment by FortuneMaster** [comment]
what the fuck is autolathe cooking?

**Comment by Wouju** [comment]
> what the fuck is autolathe cooking?

printing food i guess

i mean we already do that with Neotheo disks

**Comment by FortuneMaster** [comment]
> > what the fuck is autolathe cooking?
> 
> printing food i guess
> 
> i mean we already do that with Neotheo disks

Yeah, NT can do that, but they also posess the unique autolathe and technology to pull such stuff off. On the other hand I believe normal autolathe has just toolbox instruments. If such stuff is even needed, there should be some separate machine for that to make sense.

**Comment by genessee-forgot-his-password-again** [comment]
> what the fuck is autolathe cooking?

It makes packaged food out of raw sugar or nutriment. The way I see it, the autolathe is able to take the reagents you give it and basically just wrap them in plastic packaging. You're not really cooking anything - like I said, you're just packaging existing nutriment or sugar.

**Comment by genessee-forgot-his-password-again** [comment]
Alright I'll get back to this once I figure out mapmerging. Having some issues with it, plus have real life stuff to deal with.

**Pull request closed without merging**